src/compas_fab/ghpython/components/Cf_VisualizeRobot/code.py

```diff
@@ -89,7 +90,13 @@ def RunScript(
 
                 if update_scene:
                     scene = PlanningScene(robot)
-                    scene = robot.client.get_planning_scene()
+                    try:
+                        scene = robot.client.get_planning_scene()
+                    except BackendFeatureNotSupportedError:
+                        print("The selected backend does not support displaying collision meshes. Feature disabled.")
```

Comment:
I think helpful error messages should follow a pattern that is roughly:

- Explain clearly that there is a problem
- Explain what the problem is
- If possible, provide a solution so that the user can complete the process, or
- Point them to where they can go for help
- Make a frustrating scenario as pleasant as possible

In this case, I am not sure what you mean by "Feature disabled." Which feature? Is the whole Visualize Robot component being disabled? Is the collision meshes being disabled? (aka. Do I still have the CMs in the backend?) 
And then what I can do about this? How can I visualize the CMs? Maybe the backend doesn't support CMs in the first place and there are nothing to show. Maybe the backend has CMs, and we just cannot get them out. Maybe I should try another backend that supports the feature.

Reply:
good point 👍  fixed at